_layouts/song_item.html

```diff
@@ -5,8 +5,22 @@
 meta:
   - label: 'Title'
     value: page.label
-  - label: 'Ukrainian'
-    value: page.ukrainian-title
+  - label: 'Composer'
```

Comment:
[This line](https://github.com/womenofbandura/women-of-bandura/pull/22/files#diff-1eccffb1a5c5ebe79a9871e499479da702ac750a3284e26773cada3f9ebb2d6dL14) at the top of the metadata template is _supposed_ to strip out values that are empty strings... but I don't know why it's not working in the case of this song metadata! 

Any ideas?

Reply:
Figured it out - for whatever reason, that line of code wouldn't strip cells with ONLY whitespace " ", which doesn't equal "". I went through and got rid of all the " " in the csv, and lo and behold it works.

Reply:
Oh, interesting! Thanks!